server/www/templates/views/list.html

```diff
@@ -18,9 +18,9 @@
                 </div>
             </div> -->
 
-            <!-- <button type="button" class="button-secondary alt icon-only-when-small export">
-                <span>Export</span>
-            </button> -->
+            <button type="button" class="button-secondary alt icon-only-when-small export" ng-click="exportPosts()">
+                <span translate>post.post_actions.export</span>
+            </button>
```

Comment:
Probably not. For now it's positioned with the other bulk data actions but I should disable the check.